lib/touchbar.es

```diff
@@ -92,13 +92,8 @@ export const touchBar = new TouchBar({
   escapeItem: poibutton,
 })
 //Change Volume btn
-export const touchBarreinit = (muted) => {
-  if (muted) {
-    volume.icon = path.join(ROOT, 'assets', 'img', 'touchbar', 'volume-off.png')
-  }
-  else {
-    volume.icon = path.join(ROOT, 'assets', 'img', 'touchbar', 'volume-up.png')
-  }
+export const touchBarreinit = () => {
```

Comment:
verb + noun is good for general-purpose functions. In this case, these functions are designed for touch bar specifically. It is more like  `gameRefreshPage` to emphasize the object it deal with